testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksInsertCompatibility.java

```diff
@@ -130,6 +133,40 @@ public void testPartitionedInsertCompatibility()
         }
     }
 
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_OSS, PROFILE_SPECIFIC_TESTS})
+    public void testPartitionedInsertTimestampTZCompatibility()
+    {
+        String tableName = "test_dl_partitioned_insert_timestampTZ" + randomNameSuffix();
+        onTrino().executeQuery("" +
+                "CREATE TABLE delta.default." + tableName +
```

Comment:
Can Spark read the content we inserted with Trino?

Reply:
add the spark sql to verify the result, but spark column type is timestamp(not timestamp with time zone)

Reply:
That's fine. Delta uses timestamp_ntz for timestamp **_without_** time zone type.